lego/apps/events/models.py

```diff
@@ -84,7 +84,7 @@ def admin_register(self, user, pool, admin_reason, feedback=''):
         :param feedback: Feedback to organizers
         :return: The registration
         """
-        if self.pools.filter(id=pool.id).exists():
+        if not pool or self.pools.filter(id=pool.id).exists():
```

Comment:
What do you think about doing 
```
if pool and not self.pools.filter(id=pool.id).exists():
    raise NoSuchPool()
with transaction.atomic():
    ...
```?
Nitpicking, but I like avoid indenting the logic like this